pkg/autoscaler/autoscaler.go

```diff
@@ -198,7 +198,7 @@ func (a *Autoscaler) Scale(ctx context.Context, now time.Time) (int32, bool) {
 	logger.Debugf("STABLE: Observed average %0.3f concurrency over %v seconds.", observedStableConcurrency, config.StableWindow)
 	logger.Debugf("PANIC: Observed average %0.3f concurrency over %v seconds.", observedPanicConcurrency, config.PanicWindow)
 
-	isOverPanicThreshold := observedPanicConcurrency/readyPods >= target*2
+	isOverPanicThreshold := observedPanicConcurrency/float64(readyPodsCount) >= target*2
```

Comment:
This is no longer guarded against division by zero, is it?

Reply:
Good caught. Added the zero guard.